draft-wood-tls-external-psk-importer.md

```diff
@@ -13,6 +13,11 @@ stand_alone: yes
 pi: [toc, sortrefs, symrefs]
 
 author:
+ -
+       ins: D. Benjamin
+       name: David Benjamin
+       organization: Google, LLC.
+       email: davidben@google.com
```

Comment:
I feel like I haven't really done anything yet to be considered an author, but I don't object. :-)